sinc/octave_full.sinc

```diff
@@ -0,0 +1,12 @@
+#include "octave_minimal.sinc"
+
+apps:
+{
+    $CURDIR/../apps/octave/io.${LEGATO_TARGET}.app
```

Comment:
Can we use an `$OCTAVE_ROOT` environment variable here?
That would give more flexibility on how people structure their workspace directories, which can make a big difference when integrating with their own SCM tools and procedures.  In particular, I expect I'll need that when building leaf packages for our stuff.

Reply:
Good suggestion

Reply:
done